resources/serverless/values.yaml

```diff
@@ -73,6 +73,12 @@ images:
   functionRuntimeNodejs12:
     repository: "eu.gcr.io/kyma-project/function-runtime-nodejs12"
     tag: "1cc47db6"
+  functionRuntimeNodejs10:
```

Comment:
I wonder if such construction is more readable:
```
  functions:
    runtimes:
      nodejs12:
        repository: "eu.gcr.io/kyma-project/function-runtime-nodejs12"
        tag: "1cc47db6"
      python37:
        repository: "eu.gcr.io/kyma-project/function-runtime-python37"
        tag: "1cc47db6"
```

Reply:
probably it is :D gimme a second